README.md

```diff
@@ -14,6 +14,7 @@ You no need to write any styles or HTML code in this task. Change only `main.js`
 Hints: 
 - Field has `wall` class.
 - Spider image has `spider` class.
+- Use DOM element's properties for calculating spider position.
```

Comment:
@marina-tilniak mb better `Do not assign css properties via js - we know that you know how to do it 😉. Use DOM element's properties for calculating spider position instead.`

Reply:
to make it not a recommendation, but prohibit solving task via css

Reply:
feel free to rephrase it 🙃

Reply:
i like the shorter version :) we can add `Do not assign css properties via js` to the end for clarification